Dotfiles/claude/README.md

```
@@ -7,14 +7,14 @@ repo and is deliberately not synced.
 
 ## Code review
 
-`/wtf-review-changes` reviews recent changes in a **fresh context** — a subagent
+`/wtf-code-review` reviews recent changes in a **fresh context** — a subagent
 that never sees the conversation which wrote the code, so it cannot inherit the
 author's assumptions about it.
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)